Popup/Popup.js

```diff
@@ -57,33 +57,46 @@ const PopupBase = kind({
 		onClose: PropTypes.func,
 		onHide: PropTypes.func,
 		open: PropTypes.bool,
+
+		/**
+		 * Sets the position of the popup on the screen.
+		 *
+		 * @type {('bottom'|'center'|'fullscreen'|'left'|'right'|'top')}
+		 * @default 'center'
+		 * @public
+		 */
+		position: PropTypes.oneOf(['bottom', 'center', 'fullscreen', 'left', 'right', 'top']),
```

Comment:
Code looks we only support `center` and `top` right? then let's remove rest

Reply:
I removed the extra values